k9mail/src/main/java/com/fsck/k9/service/MailService.java

```diff
@@ -88,7 +89,7 @@ public void onCreate() {
 
     @Override
     public int startService(Intent intent, int startId) {
-        long startTime = System.currentTimeMillis();
+        long startTime = SystemClock.elapsedRealtime();
```

Comment:
Line #152 uses currentTimeMillis, which means this is now broken after the rebase.